docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[searchable-snapshots]]
+== {search-snaps-cap}
+
+Nodes in a distributed system like {es} will inevitably fail from time to time.
+To protect your data against node failures, by default when you index a
+document into {es} it is stored on two or more nodes. You also take periodic
+<<snapshot-restore,snapshots>> of your data so that you can recover from more
+serious failures. This means that each document is stored in at least three
+places. These extra copies are important for resiliency, but the storage they
+consume has an impact on your cluster's operating costs. The two storage
+mechanisms have different, but complementary, performance characteristics:
+
+* Snapshot repositories are much more reliable than local storage on individual
+  nodes.
+
+* The monetary cost per GB stored in a snapshot repository is usually lower
+  than on a node.
+
+* The monetary cost per read or write operation on a snapshot repository is
+  usually much higher.
+
+* Reading or writing data in a snapshot repository usually takes much more time
+  compared with accessing a node's local storage.
+
+{search-snaps-cap} let you reduce your operating costs by treating the snapshot
+as the authoritative copy of some of your indices. The high reliability of the
+snapshot repository removes the need to keep multiple copies of their data in
+your cluster purely for resiliency. {es} makes a copy of a searchable snapshot
+on the nodes in the cluster to reduce the performance impact and costs of
+accessing the snapshot repository.
+
+With {search-snaps-cap} you may be able to halve your cluster size without
+increasing the risk of data loss or reducing the amount of data exposed to
+searches. Put differently, {search-snaps-cap} may allow you to expose twice as
+much data to searches for a given cluster size.
+
+=== Using searchable snapshots
+
+A searchable snapshot can be searched just like any other index.
+{search-snaps-cap} are often used to access a large archive of historical data,
+for which searches may sometimes be complex and time-consuming.
```

Comment:
I wonder if we are here framing searchable snapshots as being slow, which is not necessarily the case. I think I get that the time span of the search will be large and this adds to query time, hence async search is necessary, but I would like to move that to the meat of the section more than this introducing paragraph.

Reply:
I was trying not to, but yes it still suggests slowness. I moved this to a `TIP` in 8beb64bc81451f5953bdb8c1a2626857e23ac8e3 and replaced this sentence with one indicating that performance should be similar to a regular index.